src/utils/notifications/requestNotification.js

```diff
@@ -59,6 +72,23 @@ class Notifications {
       }
     });
   }
+
+  /**
+   * @return {function} create notifications
+   */
+  async requestUpdated() {
+    await emitter.on('request update', async (request) => {
+      let notificationMessage;
+      let ownerId;
+      const { userId } = await this.makeNotification(request);
+      if (request.status !== 'Pending') {
+        notificationMessage = `Your Request has been ${request.status.toLowerCase()}`;
+        ownerId = userId;
+      }
+
+      this.notify(notificationMessage, ownerId, 'updated', request.id);
```

Comment:
Shouldn't this line be inside the if statement above

Reply:
I left it out cause there is another feature which is to use the same method and it does not use the check